cartoframes/dataset.py

```diff
@@ -8,6 +8,7 @@
 
 from carto.exceptions import CartoException, CartoRateLimitException
 from .geojson import load_geojson
+from .dataset_info import DatasetInfo
```

Comment:
do we need a package only for this class? Maybe we can put it inside `datasets`

Reply:
I feel more comfortable using a different file for each class but it is only a personal preference from other languages (maybe not for Python)

Reply:
It's quite usual having several classes in one module. From the developer point of view, I would expect having everything related to datasets inside `cartoframes.datasets.*`

We could do it by just adding the class to the `datasets.py` file or it could be handled with a package, creating a folder with both files and an __init.py__ file

Reply:
Ok, I will put them together, but we should split into several files in a module in the future. I don't want to do it right now, because we will need to take some decisions out of scope in this PR